packages/augur-ui/src/modules/common/selection.tsx

```diff
@@ -390,6 +401,50 @@ export const PillSelection = ({
   );
 }
 
+export const Tab = ({
```

Comment:
@jfarid27 Is this PR moving in the right direction? Active and disable state are still in progress. I will add it soon

Reply:
@rubyweb3 I'm not sure. I actually haven't looked at this in a while. Which issue are you referring to?

Reply:
Sorry. @ wrong person